proposals/lip-0051.md

```diff
@@ -2575,6 +2575,7 @@ This introduces a different token handling mechanism for the whole Lisk ecosyste
 
 TBA
 
+[lip-0037#chain-identifiers-1]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#chain-identifiers-1
 [lip-0037#getMainchainID]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#mainchain-chain-id
 [lip-0043]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0043.md
```

Comment:
This link can now be removed

Reply:
Removed. Removed one more link that was not needed.